### PR TITLE
Create Maryland.MD

### PR DESCRIPTION
**Pull request by Freezman13**

### CODE REVIEW

File: reports/Maryland.MD [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Prince George County | (Believed to be) June 2nd.
```
Comment (samgoldman) [inline reports/Maryland.MD:1]: Shouldn't the date be after the incident title?

Comment (amardeshbd) [reply reports/Maryland.MD:1]: Also, I am curious as to what happens when `(Believed to be) June 2nd.` data gets merged, how does the API handle this input? Is it able to properly parse data and generate `2020-06-02`?

File: reports/Maryland.MD [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+### Law enforcement beat a man on the ground.
+
+Police officer arrests a man by punching and wrestling him to the ground. After another officer arrives, they handcuff the man and kick him while he's on the ground not resisting.
+
```
Comment (emarcey) [inline reports/Maryland.MD:6]: Need to add id & tags. Especially "non-protest" tag

**Comment by emarcey** [comment]
I think we should just close this @jmc91

**Pull request closed without merging**